### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by amyblais**
- Add known issue
- Add a note that customers running an older Playbooks release need to upgrade their Playbooks instance to at least v1.26 to take advantage of channel actions functionality available from Mattermost v6.6
- Clarified that Calls is in closed beta

### CODE REVIEW

File: source/install/self-managed-changelog.md [hunk 1]
```diff
@@ -123,6 +123,7 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
  - Added ``@tippyjs/react``, ``react-popper``, ``react-slidedown`` and ``smooth-scroll-into-view-if-needed`` , and removed ``prettier`` and ``xregexp`` from https://github.com/mattermost/mattermost-webapp.
 
 ### Known Issues
+ - Oauth 2.0 applications may disappear after upgrade to v6.6.0 [MM-43500](https://mattermost.atlassian.net/browse/MM-43500).
```
Comment (cwarnermm) [inline source/install/self-managed-changelog.md:126]:
```suggestion
 - Oauth 2.0 applications may disappear after upgrading to v6.6.0 [MM-43500](https://mattermost.atlassian.net/browse/MM-43500).
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @amyblais. One small suggestion. Approving for expediency.

**Review by cwarnermm — APPROVED** [review]
Thanks, @amyblais!

**Comment by svelle** [comment]
Looks good @amyblais, do we want to remove the link entirely?

**Comment by amyblais** [comment]
Sorry I misunderstood, I've re-added the link.

**Review by svelle — APPROVED** [review]
LGTM!

**Pull request merged**